gotechnica/platform-2020
# Samuelhoward/login and registration tracking

# Description

Modifies `add_user` and `invite_user` in the users service to add items to the new `platform-<STAGE>-activity` table. When the `add_user` backend endpoint is invoked, an item is added to the `platform-<STAGE>-activity` table with REGISTER as the `event` parameter. When the `invite_user` backend endpoint is invoked, an item is added to the `platform-<STAGE>-activity` table with LOGIN as the `event` parameter. All items in the `platform-<STAGE>-activity` table also have an `id` parameter and a `user_id` parameter that is the id of the user being added or invited. In order to achieve this functionality, the `platform-<STAGE>-activity` table is added to the config files, the users service `service.yml` file, and the `create-tables.js` file in `backend/infrastructure/`.

# Steps to test

If you invoke `add_user` or `invite_user`, you should see corresponding items being added to the `platform-<STAGE>-activity` table. If you run the test suite, you should see some items added to `platform-test-activity` as the testing suite invokes both `add_user` and `invite_user`.
